src/components/button/button-test.stories.mdx

```diff
@@ -113,7 +113,7 @@ export const ButtonAsASiblingStory = ({
 export const generateButtons = (buttonType, iconPosition) => {
   return (
     <>
-      {["", ...ICONS].map((iconType) => {
+      {["", "bin"].map((iconType) => {
```

Comment:
What would you say to having the `services` icon in this snapshot too so we can have a visual test to cover this code block?
https://github.com/Sage/carbon/blob/master/src/components/button/button.style.ts#L10-L13

```suggestion
      {["", "bin", "services"].map((iconType) => {
```

Reply:
I think that line is added because the `services` icon is broken and it needs to be changed. This case is already tested in a dedicated unit test, both for the `Icon` and the `Button` component.
In my opinion, we shouldn't render that icon here, as it will result in rendering more than twenty buttons more.